docs/pages/guides/tutorials/wordpress-mysql-persistentvolume.md

```diff
@@ -62,41 +62,40 @@ These YAML files contain the Service definitions, Persistent Volume Claims, and
 
    - Select `Kubernetes Manifest` from the "Design Type" dropdown menu.
 
-        ![Import modal](./screenshots/wp1.png)
+        ![Import modal](./screenshots/wordpress-mysql-persistentvolume/wp1.png)
```

Comment:
I feel that these folder names are too long. @leecalcote should we use an ID for each tutorial somewhere and then have folder names with that ID?

Reply:
Yes sir that'll be better I guess. I've made some changes as @leecalcote sir told. If further changes are needed let me know. Thank you!